src/services/base/base.service.js

```diff
@@ -2238,12 +2446,19 @@ module.exports.graph = async (args, {req}, resource_name, collection_name) => {
         /**
          * @type {[{resource: Resource, fullUrl: string}]}
          */
-        const uniqueEntries = entries.reduce((acc, item) => {
+        let uniqueEntries = entries.reduce((acc, item) => {
             if (!acc.find(a => a.resourceType === item.resource.resourceType && a.id === item.resource.id)) {
                 acc.push(item);
```

Comment:
maybe move the filter below into this function to not have to loop twice

Reply:
good point

Reply:
resolved